src/The_Cartridge_Header.md

```diff
@@ -127,145 +137,160 @@ Code | Publisher
+Various "PD" ROMs ("Public Domain" homebrew ROMs, generally tagged with `(PD)`
+in the filename) are known to use the `$01` RAM Size tag, but this is believed
 to have been a mistake with early homebrew tools and the PD ROMs often don't use
 cartridge RAM at all.
 
-## 014A - Destination Code
+## 014A – Destination Code
+
+This byte specifies whether this version of the game is intended to be sold in
+Japan or elsewhere.
 
-Specifies if this version of the game is supposed to be sold in Japan,
-or anywhere else. Only two values are defined.
+Only two values are defined:
 
-- $00: Japanese
-- $01: Non-Japanese
+Code  | Destination
+------|------------
+`$00` | Japan
+`$01` | Other
 
-## 014B - Old Licensee Code
+## 014B – Old Licensee Code
 
-Specifies the games company/publisher code in range $00-FF. A value of
-$33 signals that the New Licensee Code (in header bytes \$0144-0145) is
-used instead. (Super Game Boy functions won't work if \<\> \$33.) A list
-of licensee codes can be found
+This byte is used in older (pre-SGB) cartridges to specify the developer/publisher
+of the game. The value `$33` indicates that the New Licensee Code (in header bytes
+`$0144`-`$0145`) is used instead. (Super Game Boy functions will only work if the
+value is `$33`.)
+
+A list of Old Licensee Codes can be found
 [here](https://raw.githubusercontent.com/gb-archive/salvage/master/txt-files/gbrom.txt).
 
-## 014C - Mask ROM Version number
+## 014C – Mask ROM Version number
 
-Specifies the version number of the game. That is usually $00.
+This byte specifies the version number of the game. The value is usually `$00`.
 
-## 014D - Header Checksum
+## 014D – Header Checksum
 
-Contains an 8 bit checksum across the cartridge header bytes $0134-014C.
-The boot ROM computes `x` as follows:
+This byte contains an 8-bit checksum computed from the cartridge header bytes
+`$0134`-`$014C`. The boot ROM computes the checksum as follows:
 
+```c
+checksum := 0
+address  := 0x0134
+while address <= 0x014C:
+	checksum := checksum - [address] - 1
+	address  := address + 1
 ```
-x = 0
-i = $0134
-while i <= $014C
-	x = x - [i] - 1
-```
 
-If the byte at $014D does not match the lower 8 bits of `x`, the boot ROM will lock up,
-and the cartridge program **won't run**.
+The boot ROM verifies this checksum. If the byte at `$014D` does not match the
+lower 8 bits of `checksum`, the boot ROM will lock up, and the program on the
+cartridge **won't run**.
+
+## 014E-014F – ROM Checksum
 
-## 014E-014F - Global Checksum
+These bytes contain a 16-bit (big-endian) checksum simply computed as the sum of
+all the bytes of the cartridge ROM (except these two checksum bytes).
 
-Contains a 16 bit checksum (upper byte first) across the whole cartridge
-ROM. Produced by adding all bytes of the cartridge (except for the two
-checksum bytes). The Game Boy doesn't verify this checksum.
+The boot ROM does not verify this checksum.
```

Comment:
Why the change from "Game Boy" to boot rom?

Reply:
Because it's more precise, honestly. Though you are right that for a general negation, the suggested passive voice is better.

Reply:
Again what @ISSOtm said - just stating that the checksum is verified by the Game Boy is pretty ambiguous. While the exact details of the boot procedure are outside the scope of this document, I don't think it's bad to simply mention that it is the boot ROM that does the checksum checking.